roles/loki/tasks/main.yml

```diff
@@ -20,4 +20,4 @@
     release_namespace: "{{ loki_helm_release_namespace }}"
     create_namespace: true
     kubeconfig: /etc/kubernetes/admin.conf
-    values: "{{ _loki_helm_values | combine(loki_helm_values, recursive=True) }}"
+    values: "{{ _loki_helm_values | combine(loki_helm_values, recursive=True) | to_json }}"
```

Comment:
What is the reason behind the `to_json` here since it seems all other roles don't need it?

Reply:
Add this because loki raise error when deploy helm chart:
``` error unmarshaling JSON: while decoding JSON: json: cannot unmarshal string into Go value of type map[string]interface {} ```